.github/ISSUE_TEMPLATE/question.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Question
-about: Ask a question about how the software works, or how to use it
+about: Ask for help with anything related to OSRD
```

Comment:
Maybe keep some more specifics while opening the possibilities : 

“Ask us for help with how the software works, how to use it or anything related to OSRD”